src/services

```diff
@@ -357,8 +357,9 @@ p({
   "name": "httpBlobService",
   "authenticate": false,
   "serviceScript": """
-    httpBlobService = new foam.nanos.blob.HttpBlobService(x, x.get("blobStore"));
-    return new foam.nanos.http.SessionWebAgent("service.run.httpBlobService", httpBlobService);
+    return new foam.nanos.blob.HttpBlobService(x, x.get("blobStore"));
```

Comment:
@Mykola97 let's leave httpBlob and httpFileService requiring login/authentication, and use service/file for unauthenticated requests.  So we'll revert these two changes.